pom.xml

```diff
@@ -124,9 +124,9 @@
 			<version>${dropwizard.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.12</version>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
```

Comment:
I’m somewhat unfamiliar with JUnit 5. But from what I read, should this be `junit-jupter` instead of `junit-jupiter-engine`?

Reply:
hi @JonMR ,
junit-jupiter-engine contains junit-jupiter-api that is required to write tests, engine is needed to run junit 5 tests (https://search.maven.org/artifact/org.junit.jupiter/junit-jupiter-engine/5.5.2/jar)

junit-jupiter contains the same things that are mentioned above + junit-jupiter-params that can be used for parameterised tests. Because now in the project there are no parameterised tests I think this is not needed now. (https://search.maven.org/artifact/org.junit.jupiter/junit-jupiter/5.5.2/jar)